### PR TITLE
⭐️ support filtering discovered github repos

### PR DESCRIPTION
**Pull request by imilchev**
We can now filter out github repos using globs:
```
cnquery scan github org podkrepi-bg --repos-exclude api,frontend
```

```
cnquery scan github org podkrepi-bg --repos api,frontend
```

### CODE REVIEW

File: providers/github/resources/discovery.go [hunk 1]
```diff
@@ -200,3 +206,55 @@ func getMqlGithubUser(runtime *plugin.Runtime, userName string) (*mqlGithubUser,
...
+	return nsFilter
+}
+
+func (f *ReposFilter) skipRepo(namespace string) bool {
+	// anything explicitly specified in the list of includes means accept only from that list
+	if len(f.include) > 0 {
+		for _, ns := range f.include {
+			g, err := glob.Compile(ns)
+			if err != nil {
+				log.Error().Err(err).Msg("failed to compile glob")
```
Comment (czunker) [inline providers/github/resources/discovery.go]: nit: I think we should mention that a glob for a repo name failed.

File: providers/github/resources/discovery_test.go
Comment (czunker) [inline providers/github/resources/discovery_test.go:1]: Could you please also add a test where both options are specified?

Comment (imilchev) [reply providers/github/resources/discovery_test.go:1]: added a test

**Review by czunker — APPROVED** [review]
Thanks @imilchev

**Pull request merged**